tests/social-share-button.test.js

```
@@ -0,0 +1,568 @@
+
+    instance.closeModal();
+    // closeModal has a 200ms timeout
+    await new Promise((resolve) => setTimeout(resolve, 250));
+
+    expect(document.activeElement).toBe(instance.button);
+  });
+
+  it("should trap focus on Tab at the last element", async () => {
+    const instance = createInstance();
+    instance.openModal();
+    await new Promise((resolve) => setTimeout(resolve, 20));
```

ESLint

[error] 'setTimeout' is not defined.

(no-undef)